src/methods/receiver.md

```diff
@@ -13,3 +13,5 @@ are other possible receivers for a method:
   transmitted.
 * No receiver: this becomes a static method on the struct. Typically used to
   create constructors which are called `new` by convention.
+
+There are even more types, e.g., `Box<Self>`.
```

Comment:
Consider referencing https://doc.rust-lang.org/reference/special-types-and-traits.html so the class can see the enumeration of special receiver types?  (Rc, Arc, Pin are the remaining ones for the moment).

Reply:
Great idea to include a link to that page.